src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -72,9 +86,12 @@
 
     @Test
     public void getStatisticAboutBananaRepeatedMethodCall() {
+        try {
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
-
+        } catch (FileNotFoundException e) {
+            System.out.println("Can't write data to file");
```

Comment:
The error message 'Can't write data to file' is not very informative. Consider using a more descriptive message that includes the exception details.